confgenerator/files.go

```diff
@@ -64,11 +68,6 @@ func (uc *UnifiedConfig) GenerateFilesFromConfig(ctx context.Context, service, l
 	case "otel":
 		// Fetch resource information from the metadata server.
 		var err error
-		MetadataResource, err = resourcedetector.GetResource()
```

Comment:
This needs to stay, for now, for the benefit of the prometheus receiver (until that receiver is fixed).